docs/tutorials/own_machine/01_create_project.rst

```diff
@@ -0,0 +1,32 @@
+.. _create_project_own_machine:
+
+Create a new project
+--------------------
+
+First, head to renkulab.io_ (or your own instance of
```

Comment:
why not just install `renku` and run `renku init`?

Reply:
After you do `renku init`, how do you connect the repo back to RenkuLab.io (or whichever renku platform instance), so that your project is publicly visible? I'm not finding documentation about this in readthedocs. 

I'm sure it can be done, but so far creating the project in RenkuLab.io/whichever platform and then `renku clone` seems simpler...?

Reply:
actually you can just add a remote and push to gitlab - as long as the repo doesn't exist it will create one. Not sure if that is or isn't documented... probably isn't.